src/pages/components/Sidebar/SidebarItem.tsx

```diff
@@ -33,4 +43,9 @@ const SidebarItem: React.FC<SidebarItemProps> = function SidebarItem({ icon, hre
   );
 };
 
+SidebarItem.defaultProps = {
+  activeBgColor: 'transparent',
+  activeTextColor: 'gray.800',
+};
```

Comment:
we don't use the `defaultProps` convention, it's no longer necessary with TS - if you want default values for this component, you will need to provide these values directly to the props

Reply:
Okay, had a bug earlier, and this fixed it...removed